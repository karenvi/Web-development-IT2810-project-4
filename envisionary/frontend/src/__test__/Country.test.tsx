import { MockedProvider } from '@apollo/client/testing';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from "react-router-dom";
import { RecoilRoot } from 'recoil';
import { mocks } from './mocks/mocks';
import Country from '../pages/Country';
export {};



describe("Testing Countries component", () => {
    it("snapshot test", () => {

        const component: renderer.ReactTestRenderer = renderer.create(
            <Router>
                <MockedProvider mocks={mocks} addTypename={false}>
                    <RecoilRoot>
                        <Country />
                    </RecoilRoot>
                </MockedProvider>
            </Router>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot(); 
    });

    it('Contains the right elements', () => {

        const component: renderer.ReactTestRenderer = renderer.create(
                <Router>
                    <MockedProvider mocks={mocks} addTypename={false}>
                        <RecoilRoot>
                            <Country />
                        </RecoilRoot>
                    </MockedProvider>
                </Router>
            );

        expect(component.root.findAllByType('h1')[0].props.children).toBe('Afghanistan');

    });

});