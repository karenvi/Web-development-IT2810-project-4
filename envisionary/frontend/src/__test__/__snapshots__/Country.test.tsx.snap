// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Country component Snapshot test of country component 1`] = `
<div>
  <main
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1fmu37l-MuiPaper-root-MuiCard-root"
    style="background-color: rgb(23, 42, 58); color: white;"
  >
    <h1
      class="MuiTypography-root MuiTypography-h3 css-3pne0l-MuiTypography-root"
    >
      Afghanistan
    </h1>
    <div
      class="MuiBox-root css-1q2t9c2"
    >
      <span
        aria-label="0 Stars"
        class="MuiRating-root MuiRating-sizeMedium MuiRating-readyOnly css-ryrseu-MuiRating-root"
        role="img"
      >
        <span
          class="MuiRating-decimal css-34he1w-MuiRating-decimal"
        >
          <span
            style="width: 0%; overflow: hidden; position: absolute;"
          >
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </span>
        <span
          class="MuiRating-decimal css-34he1w-MuiRating-decimal"
        >
          <span
            style="width: 0%; overflow: hidden; position: absolute;"
          >
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </span>
        <span
          class="MuiRating-decimal css-34he1w-MuiRating-decimal"
        >
          <span
            style="width: 0%; overflow: hidden; position: absolute;"
          >
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </span>
        <span
          class="MuiRating-decimal css-34he1w-MuiRating-decimal"
        >
          <span
            style="width: 0%; overflow: hidden; position: absolute;"
          >
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </span>
        <span
          class="MuiRating-decimal css-34he1w-MuiRating-decimal"
        >
          <span
            style="width: 0%; overflow: hidden; position: absolute;"
          >
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
          <span>
            <span
              class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                data-testid="StarIcon"
                focusable="false"
                style="opacity: 0.55;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                />
              </svg>
            </span>
          </span>
        </span>
      </span>
      <span
        class="MuiTypography-root MuiTypography-overline css-1p1534q-MuiTypography-root"
      >
         
        0.00
         / 5 (
        0
        )
      </span>
    </div>
    <div
      class="MuiBox-root css-105u4yi"
    >
      <div
        class="recharts-responsive-container"
        style="width: 100%; height: 100%;"
      />
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-y1kof7-MuiContainer-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oyhdl6-MuiContainer-root"
      >
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            Population rank:
          </b>
           
          36
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            Country code:
          </b>
           
          AFG
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            Capital:
          </b>
           
          Kabul
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline MuiTypography-alignLeft css-2t51qb-MuiTypography-root"
        >
          <b>
            Continent:
          </b>
           
          Asia
        </span>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oyhdl6-MuiContainer-root"
      >
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            Area:
          </b>
           
          652 230
           km²
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            Density:
          </b>
           
          63.0587
           per km²
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline css-b3fch5-MuiTypography-root"
        >
          <b>
            GDP growth rate:
          </b>
           
          1.0257
        </span>
        <span
          class="MuiTypography-root MuiTypography-overline MuiTypography-alignLeft css-2t51qb-MuiTypography-root"
        >
          <b>
            Percentage of world population:
          </b>
           
          0.52
          %
        </span>
      </div>
    </div>
    <div
      class="MuiBox-root css-jhhoj"
    >
      <h2
        class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
      >
        Reviews of 
        Afghanistan
        :
      </h2>
    </div>
    <div
      class="MuiBox-root css-jhhoj"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-142aol9-MuiButtonBase-root-MuiButton-root"
        id="button-write-review"
        tabindex="0"
        type="button"
      >
        Review 
        Afghanistan
        <span
          class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="EditIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 0 0-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
    <div
      class="MuiBox-root css-1wir79b"
    >
      <div
        class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded css-1ab4lby-MuiPaper-root"
        style="background-color: rgb(30, 55, 76); color: rgb(255, 255, 255);"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-1ebqi03-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-8 css-gqqy07-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
            >
              Hans og Grethe
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 css-aykyki-MuiTypography-root"
            >
              rated it
            </p>
            <span
              aria-label="5 Stars"
              class="MuiRating-root MuiRating-sizeMedium MuiRating-readyOnly css-ryrseu-MuiRating-root"
              role="img"
            >
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
            </span>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-4 css-z2noyp-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignRight css-14h8uwo-MuiTypography-root"
            >
              26.10.2022, 15:53
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-zpphrz-MuiTypography-root"
            >
              Fint og fredelig land for et idyllisk opphold på all-inclusive.
            </p>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded css-1ab4lby-MuiPaper-root"
        style="background-color: rgb(30, 55, 76); color: rgb(255, 255, 255);"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-1ebqi03-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-8 css-gqqy07-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ndaqpv-MuiTypography-root"
            >
              Trond
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 css-aykyki-MuiTypography-root"
            >
              rated it
            </p>
            <span
              aria-label="3.5 Stars"
              class="MuiRating-root MuiRating-sizeMedium MuiRating-readyOnly css-ryrseu-MuiRating-root"
              role="img"
            >
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 50%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconFilled css-7qmtgc-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      style="opacity: 0.55;"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
              <span
                class="MuiRating-decimal css-34he1w-MuiRating-decimal"
              >
                <span
                  style="width: 0%; overflow: hidden; position: absolute;"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      style="opacity: 0.55;"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty css-1c99szj-MuiRating-icon"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
                      data-testid="StarIcon"
                      focusable="false"
                      style="opacity: 0.55;"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                </span>
              </span>
            </span>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-4 css-z2noyp-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignRight css-14h8uwo-MuiTypography-root"
            >
              26.10.2022, 15:50
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-alignLeft css-zpphrz-MuiTypography-root"
            >
              Flott land å feriere i.
            </p>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
